sixtyfps_runtime/corelib/Cargo.toml

```diff
@@ -51,6 +51,7 @@ unicode-segmentation = "1.8.0"
 # Note: the rgb version is extracted in ci.yaml for rustdoc builds
 rgb = "0.8.27"
 pin-project = "1"
+num-traits = { version = "0.2", default-features = false, features = ["libm"] }
```

Comment:
do we really want to enable the libm feature unconditionally?  maybe this dependency should be optional